### ISSUE TITLE
no-unused-vars false positives when using labels

### ISSUE THREAD

**Issue by mourner** [eslint/eslint#1741]
```js
var foo = 5;

label: while (true) {
  console.log(foo);
  break label;
}
```

Will output `foo is defined but never used`. Without the label, everything works fine.

**Comment by nzakas** [comment]
Could be an escope issue.

**Comment by nzakas** [comment]
Reported to escope: https://github.com/estools/escope/issues/33#issuecomment-72323191

**Comment by nzakas** [comment]
Working on this.
